nemo/collections/multimodal/data/videoneva/conversation.py

```diff
@@ -0,0 +1,411 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
why not reuse neva `conversation.py`?

Reply:
Good point, will reuse `nemo.collections.multimodal.data.neva.conversation`